audio/boca/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        enzo1982 boca 1.0.3 v
+github.tarball_from releases
+
+categories          audio
+platforms           darwin
+license             GPL-2
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         The component framework behind the fre:ac audio converter.
+long_description    ${name} provides unified interfaces for components like encoders, decoders, taggers \
+                    and extensions as well as code to support communication between the application \
+                    and its components.
+
+homepage            https://www.freac.org/
+
+depends_lib         port:libcdio \
+                    port:libcdio-paranoia \
+                    port:expat \
+                    port:uriparser \
+                    port:smooth
+
+use_configure       no
```

Comment:
Because of this, you need to review and implement [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler)

Reply:
that does apply to all three ports here, you might want to use the `makefile` PG as that will likely set some of the things for you correctly. But yes, Frank is right: please make sure these ports all use the correct compiler.

Reply:
@mf2k Ok, I decided to remove `use_configure no` then. The configure phase does not create any makefiles but if it helps preserving the compiler selection I will keep the configure phase.

@reneeotten I added the makefile PG to all ports. The only feature I use is `makefile.prefix_name` though.

Reply:
if you use the `makefile` PG then it will automatically set `use_configure no`; you cannot run `./configure` since it doesn't exist for these ports. It's just that the `makefile` PG attempts to set the compilers and flags correctly so that if there is a "standard" Makefile that they can be used.

Reply:
Ok after reading through makefile-1.0.tcl I think I got it. I am marking this point as resolved as the makefile PG will set the compiler envs the "macports way".